tests/basic/pool_function_test.go

```diff
@@ -59,12 +68,12 @@ var _ = Describe("{StoragePoolExpandSmoky}", func() {
 		verifyPoolSizeEqualOrLargerThanExpected(poolIDToResize, targetSizeGiB)
 	})
 
-	testName = "StoragePoolExpandDiskAuto"
-	testDescription = "Validate storage pool expansion using auto option"
-	It("select a pool that has I/O and expand it by 100 GiB with auto type. ", func() {
+	It("Select a pool that has I/O and expand it by 100 GiB with auto type. ", func() {
```

Comment:
Same comment as https://github.com/portworx/torpedo/pull/1673/files#r1336488163 for description